StRoot/StRefMultCorr/Param.h

```diff
@@ -481,10 +471,96 @@ const string mParamStr_ref1[nID_ref1][nSet_ref1] = {
     "150",    // Normalization start
     "0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00",   // Vz correction parameters (switched to new scheme, parameters defined near the shape corrections)
     "1.13418,-1.68334e-2,1.68995e-3,1.03279e-2,-4.42083e-1,0,1.47591e-3,4.36703e-1",  // Trigger efficiency
-    "2.14803e+02,-4.58703e-4" // Luminosity correction
+    "2.14803e+02,-4.58703e-4"  // Luminosity correction
+  },
+  //mParameterIndex = 41
+  //Run21 Au+Au 7.7 GeV, TrigerID = 810010,810020,810030,810040
+  {
+    "2021:7.7:22031042,22121018:-145,145",                     // Year, energy, run start end, Vz range
+    "4,6,8,12,16,21,28,37,47,59,74,91,112,135,164,201",      // Centrality definition
+    "100",    // Normalization start
+    "0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00",   // Vz correction parameters (switched to new scheme, parameters defined near the shape corrections)
+    "1.48905,-9610.65,622.081,4102.52,-1.06441e-05,0,7.59456e+07,7.24623e-11",  // Trigger efficiency
+    "0,0" // Luminosity correction
   }
 };
 
+//=======================================================================================
+//if you want to add new parameters, please always add after the current parameters, do not insert between them
+//the order of them is important!!!!!!!!!!!!!
+// FXT section
+//=======================================================================================
+const int nID_ref5  = 6;//after add N more new group of parameters, need to update nID_ref1 += N
```

Comment:
Run ID of 5.75 GeV Run 20 dataset overlaps other Run 20 FXT datasets. To separate 5.75 GeV Run 20 using two mParameterIndex.
```suggestion
const int nID_ref5  = 7;//after add N more new group of parameters, need to update nID_ref1 += N
```